src/flash/display/Sprite.ts

```diff
@@ -171,6 +171,7 @@ module Shumway.AVM2.AS.flash.display {
             var symbol: Shumway.Timeline.DisplaySymbol = null;
             if (hasCharacter) {
               symbol = <Shumway.Timeline.DisplaySymbol>loaderInfo.getSymbolById(placeObjectTag.symbolId);
+              // The Flash Player ignores references to not defined symbols here. So should we.
```

Comment:
Do we pick the symbol up on a later playback of the same frame if it became available in the meantime? The Flash Player does, AFAIR.